editor/plugins/plugin_config_dialog.cpp

```diff
@@ -109,6 +109,8 @@ void PluginConfigDialog::_on_canceled() {
 void PluginConfigDialog::_on_required_text_changed() {
 	if (name_edit->get_text().is_empty()) {
 		validation_panel->set_message(MSG_ID_PLUGIN, TTR("Plugin name cannot be blank."), EditorValidationPanel::MSG_ERROR);
+	} else if (name_edit->get_text().contains("/") || name_edit->get_text().contains("\\")) {
```

Comment:
```suggestion
	} else if (!name_edit->get_text().is_valid_filename()) {
```
I'd say, same as done with the script creation dialog

Reply:
<img width="854" alt="Screenshot 2025-03-30 at 2 44 47 PM" src="https://github.com/user-attachments/assets/009a16e7-58b5-4a07-a5f5-a4febf28af8d" />

@AThousandShips  I have added it